openbook_auth/tests/views/test_user_subscriptions.py

```diff
@@ -0,0 +1,194 @@
+import random
+from django.urls import reverse
+from faker import Faker
+from rest_framework import status
+from openbook_common.tests.models import OpenbookAPITestCase
+
+import logging
+import json
+
+from openbook_common.tests.helpers import make_user, make_authentication_headers_for_user
+
+fake = Faker()
+
+logger = logging.getLogger(__name__)
+
+
+class UserSubscriptionsAPITests(OpenbookAPITestCase):
+    """
+    UserSubscriptionsAPI Tests
+    """
+    def test_can_retrieve_user_subscriptions(self):
```

Comment:
In general, everywhere instead of `user_subscriptions`, `user_notifications_subscriptions`.